src/helper/factory/index.ts

```diff
@@ -285,13 +286,19 @@ export interface CreateHandlersInterface<E extends Env, P extends string> {
 
 export class Factory<E extends Env = any, P extends string = any> {
   private initApp?: InitApp<E>
+  #defaultOptions?: HonoOptions<E>
 
-  constructor(init?: { initApp?: InitApp<E> }) {
+  constructor(init?: { initApp?: InitApp<E>; defaultOptions?: HonoOptions<E> }) {
```

Comment:
`Factory` does not provide only `createApp()` but it provides `createMiddleware()` and `createHandlers()`. So the name `defaultOptions` may introduce confusion because the user may understand the option is not for `createApp()`. So I think it should have a different name, such as `defaultAppOptions`.

Reply:
Thank you for pointing that out. I completely overlooked the fact that it's not just about the `app`.